src/renderer/model/spotlight-layers.js

```diff
@@ -15,8 +17,25 @@ export default register => {
     : () => true
 
   const layerItem = (layerId, name, checked) => {
+
+    const exportLayer = (event) => {
+      event.stopPropagation()
+      electron.remote.dialog.showSaveDialog({ defaultPath: `layer-${layerId}.json` })
+        .then(result => {
+          if (result.canceled) return
+          try {
+            fs.writeFileSync(result.filePath, JSON.stringify(store.layer(layerId)))
+          } catch (error) {
+            console.error(error)
+          }
+        })
+        .catch(error => {
+          console.error(error)
+        })
```

Comment:
DRY: This is pretty much the same as in .../ipc/layer.js, except for two things A) I/O errors are not displayed, B) feature properties are not filtered. Stored layers have additional properties which should not be exported.
Suggestion COMMAND_EXPORT_(DEFAULT_)LAYER could be generalized to accept any layer id (default layer id is `0`).

Reply:
Arghh, did not know that ipc/layer.js exists. I'll improve the PR accordingly.